meson_options.txt

```diff
@@ -3,4 +3,5 @@ option('xwayland', type: 'feature', value: 'auto', yield: true, description: 'En
 option('x11-backend', type: 'feature', value: 'auto', description: 'Enable X11 backend')
 option('examples', type: 'boolean', value: true, description: 'Build example applications')
 option('icon_directory', description: 'Location used to look for cursors (default: ${datadir}/icons)', type: 'string', value: '')
-option('renderers', type: 'array', choices: ['auto', 'gles2'], value: ['auto'], description: 'Select built-in renderers')
+option('renderers', type: 'array', choices: ['auto', 'gles2', 'vulkan'], value: ['auto'], description: 'Select built-in renderers')
+option('xdg-foreign', type: 'feature', value: 'auto', description: 'Enable xdg-foreign protocol')
```

Comment:
Rebase gone wrong? This option has been removed from master.

Reply:
Indeed, thanks!